packages/expo-cli/src/commands/client/index.js

```diff
@@ -0,0 +1,101 @@
+import chalk from 'chalk';
+import CliTable from 'cli-table';
+import { Credentials, User, ApiV2 } from 'xdl';
+
+import urlOpts from '../../urlOpts';
+import * as appleApi from '../build/ios/appleApi';
+import { runAction, travelingFastlane } from '../build/ios/appleApi/fastlane';
+import * as credentials from '../build/ios/credentials';
+import promptForCredentials from '../build/ios/credentials/prompt/promptForCredentials';
+import selectDistributionCert from './selectDistributionCert';
+import selectPushKey from './selectPushKey';
+import generateBundleIdentifier from './generateBundleIdentifier';
+import log from '../../log';
+import prompt from '../../prompt';
+
+export default program => {
+  program
+    .command('client:ios')
+    .option(
+      '--apple-id <login>',
+      'Apple ID username (please also set the Apple ID password as EXPO_APPLE_PASSWORD environment variable).'
+    )
+    .asyncAction(async options => {
+      let authData = await appleApi.authenticate(options);
+      let user = await User.getCurrentUserAsync();
+      let bundleIdentifier = generateBundleIdentifier(authData.team.id);
+      let context = {
+        ...authData,
+        bundleIdentifier,
+        experienceName: 'Expo',
+        username: user ? user.username : null,
+      };
+      await appleApi.ensureAppExists(context);
+
+      let distributionCert = await selectDistributionCert(context);
+      let pushKey = await selectPushKey(context);
+
+      let email;
+      if (user) {
+        email = user.email;
+      } else {
+        ({ email } = await prompt({
+          name: 'email',
+          message: 'Please enter an email address to notify, when the build is completed:',
+          default: user ? user.email : undefined,
```

Comment:
I guess it will be always `undefined`.

Reply:
Uh, thanks. I think I changed the logic at some point and forgot to remove this line.